ha/core/cluster/cluster_manager.py

```diff
@@ -267,12 +267,13 @@ def shutdown(self):
 
 # Note: This class is used by version 2
 class CortxClusterManager:
-    def __init__(self):
+    def __init__(self, default_log_enable=True):
         """
         Manage cluster operation
         """
-        # TODO: Update Config manager if log utility changes.
-        ConfigManager.init("cluster_manager")
+        # TODO: Update Config manager if log utility changes.(reference EOS-17614)
+        if default_log_enable is True:
```

Comment:
Sorry, but this is more of a configmanager init to be done or not based on the argument passed. The argument name should be changed accordingly.
cc @ajay-paratmandali

Reply:
Argument name changed to "init_config_manager".

Reply:
@indrajitzagade  Akash
I think `default_log_enable` is good option 
Here, in recent change we are doing conf init both case only difference is log enabling 
so `init_config_manager` is not hold true here
```
        if init_config_manager is True:
            ConfigManager.init("cluster_manager")
        else:
            ConfigManager.init(None)
```

Reply:
@ajay-paratmandali I think I missed this recent change. And yes then the old argument name makes sense. Thanks.

Reply:
Argument name renamed to "default_log_enable".